apps/admin/backend/src/app.ts

```diff
@@ -120,6 +126,54 @@ function buildApi({
       );
     },
 
+    async setSystemSettings(input: {
+      systemSettings: string;
+    }): Promise<SetSystemSettingsResult> {
+      await logger.log(
+        LogEventId.SystemSettingsSaveInitiated,
+        assertDefined(await getUserRole()),
+        { disposition: 'na' }
+      );
+
+      const { systemSettings } = input;
+      const validatedSystemSettings = safeParseJson(
+        systemSettings,
+        SystemSettingsSchema
+      );
+      if (validatedSystemSettings.isErr()) {
+        return err({
+          type: 'parsing',
+          message: validatedSystemSettings.err()?.message,
+        });
+      }
+
+      let id;
+      try {
+        id = store.addSystemSettings(validatedSystemSettings.ok());
+        store.setCurrentSystemSettingsId(id);
+      } catch (error) {
+        const typedError = error as Error;
+        await logger.log(
+          LogEventId.SystemSettingsSaved,
+          assertDefined(await getUserRole()),
+          { disposition: 'failure', error: typedError.message }
+        );
+        return err({
+          type: 'database',
+          message: `Failed to save system settings to db. Error: ${typedError.message}`,
+        });
+      }
+
+      await logger.log(
+        LogEventId.SystemSettingsSaved,
+        assertDefined(await getUserRole()),
+        { disposition: 'success', system_settings_id: id }
```

Comment:
I believe most of our additional/custom log fields use camelCase, e.g. https://github.com/votingworks/vxsuite/blob/87da7bf630353891a84a81c39ef5ed8a9af7c538/libs/auth/src/dipped_smart_card_auth.ts#L241

Reply:
Ah sorry, wires got crossed in my brain since I think I was working on the snake-case schema before this.